src/DetectorConstruction.cpp

```diff
@@ -18,18 +19,19 @@
 #include "G4SDManager.hh"
 #include "G4LogicalVolumeStore.hh"
 
-
 DetectorConstruction::DetectorConstruction(): G4VUserDetectorConstruction()
-{}
```

Comment:
I'm a fan of the one-line `{;}` structure, put on the same line as the declaration, but what you've changed is consistent with the rest of the code.